samples/subsys/usb/mass/CMakeLists.txt

```diff
@@ -2,5 +2,9 @@ cmake_minimum_required(VERSION 3.13.1)
 include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
```

Comment:
You are missing this below before the `include` line:

```
if(BOARD STREQUAL nrf52840_pca10056)
  set(CONF_FILE nrf52840_pca10056.conf)
endif()
```

right?

I don't think the `.conf` file for a particular board is automatically picked up?

Reply:
My mistake, this was added a while back and I didn't know:
https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/app/boilerplate.cmake#L344